lib/api/conversations.js

```diff
@@ -0,0 +1,90 @@
+export const queryConversations = ({
+    Location,
+    Page = 0,
+    PageSize = 50,
+    Limit = 50,
+    LabelID,
+    Sort,
+    Desc = 1,
+    Begin,
+    End,
+    BeginID,
+    EndID,
+    Keyword,
+    To,
+    From,
+    Subject,
+    Attachments,
+    Starred,
+    Unread,
+    AddressID,
+    ID,
+    AutoWildcard
+}) => ({
+    method: 'get',
+    url: 'conversations',
+    params: {
+        Location,
+        Page,
+        PageSize,
+        Limit,
+        LabelID,
+        Sort,
+        Desc,
+        Begin,
+        End,
+        BeginID,
+        EndID,
+        Keyword,
+        To,
+        From,
+        Subject,
+        Attachments,
+        Starred,
+        Unread,
+        AddressID,
+        ID,
+        AutoWildcard
+    },
+});
+
+export const getConversation = ({ conversationID, MessageID }) => ({
+    method: 'get',
+    url: 'conversations',
+    params: { conversationID, MessageID },
```

Comment:
It should be capitalized.

Reply:
According to @bartbutler , `conversationID` should not be capitalized, reason being it's not a query string parameter, but rather a parameter in the route itself. `MessageID` on the other hand is an optional query string parameter.
I hope this is more clear in the way I rewrote the function.